py-media-storage/app/main.py

```diff
@@ -0,0 +1,67 @@
+import os
+from typing import Union
+
+# @klotho::persist {
+#   id = "fs"
+# }
+import aiofiles as fs
+from aiocache import Cache
+from fastapi import FastAPI, Response, UploadFile
+from pydantic import BaseModel
+from starlette.responses import PlainTextResponse
+
+# @klotho::persist {
+#   id = "imageKV"
+# }
+imageStore = Cache(cache_class=Cache.MEMORY)  # hello
+x = 1
```

Comment:
the comment and x=1?

Reply:
Yeah, there were a few things like that one that had accidentally made it into the old repo. I missed these.